android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/components/FormAnswer.kt

```diff
@@ -0,0 +1,78 @@
+package co.nimblehq.ic.kmm.suv.android.ui.components
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.material.TextFieldColors
+import androidx.compose.material.TextFieldDefaults
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.nimblehq.ic.kmm.suv.android.ui.theme.AppTheme
+import co.nimblehq.ic.kmm.suv.android.ui.theme.Typography
+
+typealias Placeholder = String
+
+@Composable
+fun FormAnswer(
+    placeholders: List<Placeholder>,
+    onTextChange: (Pair<Int, String>) -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        verticalArrangement = Arrangement.spacedBy(16.dp),
+        modifier = modifier
+    ) {
+        placeholders.forEachIndexed { index, placeholder ->
+            var value by remember { mutableStateOf("") }
+            val focusManager = LocalFocusManager.current
+            TextField(
+                value = value,
+                onValueChange = {
+                    value = it
+                    onTextChange(index to it)
+                },
+                singleLine = true,
+                placeholder = {
+                    Text(
+                        text = placeholder,
+                        style = Typography.subtitle1
+                    )
+                },
+                colors = formTextFieldColors(value.isEmpty()),
+                keyboardActions = KeyboardActions(onDone = { focusManager.clearFocus() }),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .size(56.dp)
```

Comment:
it seems we have a dimension for this value: defaultComponentHeight

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/120/commits/2ce21e00f1e35f205215122019f7b0a0e65747f7